components/log-viewer-webui/server/src/routes/static.ts

```diff
@@ -49,6 +49,11 @@ const routes: FastifyPluginAsync = async (fastify) => {
             prefix: "/",
             root: clientDir,
             decorateReply: false,
+            wildcard: false,
+        });
+
+        fastify.get("/streamFile", (_, reply) => {
+            reply.sendFile("index.html", clientDir);
         });
```

Comment:
to get the other React-Router defined routes in the SPA working, i believe we also need to let Fastify serve those routes with the index.html 

@coderabbitai please create an issue to track this